crates/router/src/connector/iatapay/transformers.rs

```diff
@@ -226,7 +237,11 @@ impl From<IatapayPaymentStatus> for enums::AttemptStatus {
             IatapayPaymentStatus::Failed | IatapayPaymentStatus::UnexpectedSettled => Self::Failure,
             IatapayPaymentStatus::Created => Self::AuthenticationPending,
             IatapayPaymentStatus::Initiated => Self::Pending,
-            _ => Self::Voided,
+            IatapayPaymentStatus::Tobeinvestigated
```

Comment:
@prasunna09 Can you confirm whether status mapping are right?

Reply:
<img width="821" alt="Screenshot 2023-11-13 at 2 44 41 PM" src="https://github.com/juspay/hyperswitch/assets/70575890/3899731b-cd40-4da9-b101-043183912264">
`IatapayPaymentStatus::Cleared => Self::Charged`
Can you please refer to the above screenshot shared and map the status accordingly?